internal/metamorph/server.go

```diff
@@ -174,11 +174,14 @@ func (s *Server) PutTransaction(ctx context.Context, req *metamorph_api.Transact
 	statusReceived := metamorph_api.Status_RECEIVED
 
 	// Convert gRPC req to store.StoreData struct...
+	callback := store.StoreCallback{
+		CallbackURL:   req.GetCallbackUrl(),
+		CallbackToken: req.GetCallbackToken(),
+	}
 	sReq := &store.StoreData{
 		Hash:              hash,
 		Status:            statusReceived,
-		CallbackUrl:       req.GetCallbackUrl(),
-		CallbackToken:     req.GetCallbackToken(),
+		Callbacks:         []store.StoreCallback{callback},
```

Comment:
It's a nit-pick, just an idea, but maybe it would be better to create it here directly, without creating a separate struct above. WDYT?
```suggestion
		Callbacks: []store.StoreCallback{{
			CallbackURL:   req.GetCallbackUrl(),
			CallbackToken: req.GetCallbackToken(),
		}},
```

Reply:
Done